LiskHQ/lisk-mobile
# Using QR code to sign in - Closes #278

# What was the bug or feature?
Described in #278 

### How did I fix it?

 - Moved the scanner component to a dedicated directory.
 - Passed the normalizer as a property to make it possible to have many normalizers.
 - Added the scanner functionality to Sign-in page.
 - Updated the styles.

## Type of change
- Enhancement (a non-breaking change which adds functionality)


### How to test it?
The scanner should work in both Send and Sign in pages using camera and gallery both in Android and iOS.

# Checklist:
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
